docs/pages/router/reference/testing.mdx

```diff
@@ -173,3 +180,67 @@ expect(screen).toHaveRouterState({
 ```
 
 </PaddedAPIBox>
+
+## Examples
+
+For a simple component, where you're not explicitly asserting against or invoking any `expo-router` behavior, you may simply need to render with `renderRouter` instead of `render`. If you're not testing route `pathname`s, then you can simply do this by rendering your component as the `index`:
+
+```tsx app.test.ts
+renderRouter({
+  index: jest.fn(() => <MyComponent />),
+});
+```
+
+If you wish to assert against navigation actions, you may need to mock the `router` function, as well as any other functions you wish to assert against:
+
+```tsx app.test.ts
+jest.mock('expo-router', () => ({
+  ...jest.requireActual('expo-router'),
+ useLocalSearchParams: jest.fn().mockImplementation(() => ({ // or use `mockReturnValue`
+    someParam: foo-param,
+  router: {
+    navigate: jest.fn(),
+  },
+}));
+
+...
+
+expect(router.navigate).toHaveBeenCalledWith('/profile');
+```
+
+If you're asserting against any specific `pathname` values, you'll need to change the pathname you use in `renderRouter` to whatever the pathname will actually be. This means you'll also need to use the `initialUrl` option:
+
+```tsx app.test.ts
+renderRouter(
+  {
+    '/some-path': jest.fn(() => <MyComponent />),
```

Comment:
These don't need to be wrapped in `jest.fn()` unless your asserting that the screen was rendered. I think people's tests should be asserting against their own logic and not checking if `expo-router` is rendering screens.